src/snowflake/snowpark/modin/pandas/indexing.py

```diff
@@ -1049,14 +1049,34 @@ def __getitem__(
 
         original_row_loc = row_loc  # keep a copy for error message
 
+        # IR changes! Add iloc get nodes to AST.
+        stmt = self.df._get_session()._ast_batch.assign()
+        ast = stmt.expr
+        ast.pd_dataframe_iloc_get.df.pd_dataframe_ref.id.bitfield1 = (
```

Comment:
remove the parens

Reply:
I think the linter did this when running pre-commit, I can try to get rid of it if preferred